frontend/src/Components/Map/Inputs/Hamburger.tsx

```diff
@@ -20,20 +20,36 @@ interface HamburgerProps {
  *
  * @author Lyons
  */
+
+const HEIGHT_OF_EACH_SURVEY_ITEM = 40; // Define this constant if not defined elsewhere
+
 const Hamburger: React.FC<HamburgerProps> = ({ isOpen, toggle }) => {
-  const [surveys, setSurveys] = useState<SurveyList>([]);
-  const navigate = useNavigate();
+  const [surveys, setSurveys] = useState<ISurvey[]>([]);
+  const [selectedSurvey, setSelectedSurvey] = useState<ISurvey | null>(null);
```

Comment:
If you select the survey here, how are you going to show the path on the main map?

Reply:
up